draft-irtf-cfrg-vdaf.md

```diff
@@ -2407,14 +2428,14 @@ The methods used in this computation are defined in {{prio3-auxiliary}}.
 
 This section describes the process of recovering output shares from the input
 shares. The high-level idea is that each Aggregator first queries its
```

Comment:
"proofs share" and "verifiers share" are confusing terms in this context. Don't just search and replace; we need to make sure this paragraph is still understandable.

Here and below.